pkg/syncer/spec/spec_process.go

```diff
@@ -153,7 +151,7 @@ func (c *Controller) process(ctx context.Context, gvr schema.GroupVersionResourc
 	}
 	if !exists {
 		// deleted upstream => delete downstream
-		klog.Infof("Deleting downstream GVR %q object %s/%s for upstream cluster %q", gvr.String(), upstreamNamespace, name, clusterName)
+		klog.Infof("Deleting downstream GVR %q object %s/%s for upstream cluster %q", gvr.String(), downstreamNamespace, name, clusterName)
```

Comment:
not part of autogen? (but looks like a good catch)

Reply:
Yep, sorry. Wrong commit.